demos/question-answering/README.md

```diff
@@ -23,44 +38,83 @@ To complete the tutorial follow the steps below:
   ```
   python -m ipykernel install --user --name=question-answering
   ```
-- Launch the five Notebooks in order and execute the code cells. Make sure to select the `question-answering` environment kernel for each Notebook.
+- Refresh you browser tab, so the new environment will become available.
+- Launch the five Notebooks in order and execute the code cells. Make sure to select the `question-answering`
+  environment kernel for each Notebook.
+- Use the EzUA "Import Framework" wizard to upload the tarball located inside the `application` folder to create a user
```

Comment:
you mentioned EzUA here, so I commented above to be consistent about the acronym.